web-app/js/angular/controllers/ListsController.js

```diff
@@ -39,6 +40,17 @@ profileEditor.controller('ListsEditor', function (profileService, navService, ut
         });
     };
 
+    self.loadBioStatus = function () {
+        var promise = profileService.getBioStatus(self.opusId, self.profileId);
+        promise.then(function (data) {
+            self.bioStatuses = orderBy(data, 'key');
+
+            if (self.bioStatuses.length > 0) {
+                navService.add("Bio Status", "bioStatus");
```

Comment:
We don't need a separate index item for biostatus. Change the Conservation Status item to something more generic (just "Status" for now) and make sure it is added if there are any conservation statuses OR 'bio' statuses.

(Biostatus is actually just one type of status in this new list)